modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/visualizationModes.spec.ts

```diff
@@ -54,17 +52,32 @@ test.describe('Visualization Modes in Data Set fragment', () => {
 		const SAMPLE_OBJECT_FIELD = 'dataSetToDataSetTableSections';
 		const SAMPLE_OBJECT_CHILD_FIELD = 'label';
 
+		await test.step('Update Data Set to include additionalAPIURLParameters', async () => {
+			await dataSetManagerApiHelpers.updateDataSet({
+				additionalAPIURLParameters:
+					'fields=dataSetToDataSetTableSections.label,id,label',
```

Comment:
This filters out `dataSetToDataSetTableSections` as a field in returned data? Is that why we don't need `dataSetToDataSetTableSections` object? Doesn't FDS require full object to resolve table section name `dataSetToDataSetTableSections.label`? Is this just a simple string field in returned data because it is in `fields` parameter?

Reply:
With the `fields` parameter we ask for this specific fields, and it returns the whole object:

![image](https://github.com/user-attachments/assets/e3be5c09-5d3e-4f5c-8f0d-9912b043b1ae)

Reply:
Of course, we need the `nestedFields=dataSetToDataSetTableSections` part to get the data, otherwise we will end up only with `id` and `label`. 

I've used the `fields` parameter to highlight the usage of the `additionalAPIURLParameters` and check it but only in the scope of this specific test.